packages/rust.rb

```diff
@@ -65,8 +63,13 @@ def self.install
     RUSTCOMPLETIONEOF
     IO.write("#{CREW_DEST_PREFIX}/etc/bash.d/rust", @rustcompletionenv)
     system "#{CREW_DEST_PREFIX}/share/cargo/bin/rustup completions bash > #{CREW_DEST_PREFIX}/share/bash-completion/completions/rustup"
+    Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
+      Dir.glob('*').each do |f|
```

Comment:
`Dir.glob` can used with block directly:
```ruby
Dir.glob('*') do |f|
```

Reply:
`Dir['*'] do |f|` could work also?  Less is more?

Reply:
> Dir['*'] do |f| could work also? Less is more?
As I know `Dir[]` doesn't accept block (only arrays)